### PR TITLE
backplane&14limit

### PR DESCRIPTION
**Pull request by YiHsuanZhang**
add Regular  backplane 
Not added yet Black backplane 

add 14 card conditions

### CODE REVIEW

File: src/components/MahjongSelector.tsx [hunk 1]
```diff
@@ -149,7 +160,7 @@ const MahjongSelector: React.FC = () => {
           </div>
         ))}
       </div>
-      <div>
+      <div className='tile-container'>
```
Comment (ChungNYCU) [inline src/components/MahjongSelector.tsx:162]: Can you explain where the tile-container is located?

File: src/components/MahjongSelector.tsx [hunk 2]
```diff
@@ -76,6 +84,8 @@ const MahjongSelector: React.FC = () => {
     // Check if the tile is already in the map
     const count = newSelectedTile.get(tile) || 0;
 
+    if (count === 0) return;
```
Comment (ChungNYCU) [inline src/components/MahjongSelector.tsx:88]: Why is this needed? Checking for edge cases is a good behavior, but do we need it in this case?

File: src/components/MahjongSelector.tsx (hunk 1 above)
Comment (YiHsuanZhang) [reply src/components/MahjongSelector.tsx:162]: C:\Code\TaiwanMj\taiwan-mahjong

File: src/components/MahjongSelector.tsx (hunk 2 above)
Comment (YiHsuanZhang) [reply src/components/MahjongSelector.tsx:88]: I'm not sure about this code.
Because the purpose is just to check whether the card deselection operation is valid. If this line of code is redundant, I can omit it.

File: src/components/MahjongSelector.tsx (hunk 1 above)
Comment (ChungNYCU) [reply src/components/MahjongSelector.tsx:162]: Did you push it to a branch? I can't see it.
